### PR TITLE
feat: pre-filled create index from attributes page

### PR DESCRIPTION
**Pull request by TGlide**
## What does this PR do?

Pre-fills the create an index modal when called from the attribute page

https://user-images.githubusercontent.com/26071571/203637348-8688bd84-b4c2-4f42-bced-39e2a695a18a.mp4

## Test Plan

Manual

## Related PRs and Issues

N/A

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/indexes/createIndex.svelte [hunk 1]
```diff
@@ -74,6 +74,14 @@
             });
         } finally {
             showCreateIndex = false;
+
+            // Redirect to the index page if the index is created from the attribute page
+            // P.S. goto is not working here for some reason
+            if (externalAttribute) {
+                // await goto(window.location.pathname.replace('attributes', 'indexes'), {
```
Comment (ArmanNik) [inline src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/indexes/createIndex.svelte:81]: Mmmmm strange, did you try this?  `${base}/console/project-${$page.params.project}/databases/database-${databaseId}/collection-${$collection.$id}/indexes`

**Comment by ArmanNik** [comment]
We should probably hide the menu (dropdown from clicking the 3 dots) when we open the modal :+1:

File: src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/attributes/+page.svelte [hunk 2]
```diff
@@ -92,7 +92,9 @@
                                         on:click={() => {
                                             selectedAttribute = attribute;
                                             showCreateIndex = true;
-                                        }}>Create Index</DropListItem>
+                                        }}
+                                        >Create Index
+                                    </DropListItem>
```
[the remark this answers is not in the document]
Comment (TGlide) [reply src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/attributes/+page.svelte:97]: Yeah, that seems better! I'll change it back 🙂

File: src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/indexes/createIndex.svelte (hunk 1 above)
Comment (TGlide) [reply src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/indexes/createIndex.svelte]: Yeah, it didn't work, unfortunately 😢

**Pull request merged**